.github/actions/gradle-task/action.yml

```diff
@@ -19,7 +19,7 @@ runs:
         if [[ ${{ inputs.is_release }} == "false"  ]];then
           echo "Workflow triggered by push to a branch"
         fi
-        version=${{ steps.setup_version_properties.outputs.VERSION_DISPLAY }}
+        version=${{ steps.setup_version_properties.outputs.VERSION_TAG }}
         echo "Version: $version"
-        ./gradlew ${{ inputs.gradle_command }}
+        VERSION_TAG=$version ./gradlew ${{ inputs.gradle_command }}
```

Comment:
Out of interest, why don't we do this like our other jobs?

I imagine this set-up will be more safe, in case the job uses some other env too?
```
    env:
      VERSION_BRANCH: ${{ needs.setup_version_properties.outputs.VERSION_BRANCH }}
      VERSION_COMMIT: ${{ needs.setup_version_properties.outputs.VERSION_COMMIT }}
      VERSION_DISPLAY: ${{ needs.setup_version_properties.outputs.VERSION_DISPLAY }}
      VERSION_BUILD: ${{ needs.setup_version_properties.outputs.VERSION_BUILD }}
      VERSION_TAG: ${{ needs.setup_version_properties.outputs.VERSION_TAG }}
      VERSION_LAST_TAG: ${{ needs.setup_version_properties.outputs.VERSION_LAST_TAG }}
```

Reply:
haha - I have no idea why I didn't do that originally. Probably being a bit slow towards the end of the day ;)